src/Authorization/FitOnFhir.Authorization/AuthorizationFunction.cs

```diff
@@ -33,7 +31,7 @@ public class AuthorizationFunction
 
         [FunctionName("api")]
         public async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "{p1?}/{p2?}/{p3?}")] HttpRequest req,
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "{p1?}/{p2?}")] HttpRequest req,
```

Comment:
Changes in this class are just general cleanup, and did not contribute to the callback issue.